tests/fixed/superfluous-naming.php

```diff
@@ -12,6 +12,10 @@ class FooException extends RuntimeException
 {
 }
 
+class FooError extends Error
+{
+}
+
```

Comment:
This is superfluous but not reported.. i guess there is no existing rule for this